exportdb/views.py

```diff
@@ -62,6 +64,7 @@ def get_exporter_class(self):
     def get_context_data(self, **kwargs):
         context = super(ExportView, self).get_context_data(**kwargs)
         context['title'] = _('Export database')
+        context['jquery_in_vendor'] = (parse_version(django.get_version()) >= parse_version('1.9.0'))
```

Comment:
it's generally nicer to put all the Django-version specific stuff in a `compat.py` file and import from there. Can you move this check/function to such a module?